README.md

```diff
@@ -58,6 +58,8 @@ MyApplication::Application.config.session_store :redis_store, {
 }
 ```
 
+NOTE: The sessions store uses a global lock to ensure operations are atomic. No atomic guarantees can be made if you're running multiple instances of your app, so it is recommended that you disable this by using the `threadsafe: false` option.
```

Comment:
@tubbo Good point about not describing what the config does.

Yeah, "recommended" is maybe the wrong way to put it. That said, if someone is running more than one process of rails, the mutex doesn't do anything besides slow you down (since it can't lock between processes). If you're running jruby then it _really_ slows you down. So from my understanding it really only works if you only run a single rails process. Maybe we _should_ make it default?

Or maybe it should say something like:
> "The sessions store uses a global lock to ensure operations are atomic. If you are running multiple processes, no atomic access guarantees can be made. You may want to consider setting the `threadsafe: false` option, which disables any attempt to protect the session store from current access. If you have disabled `Rack::Lock` middleware in your app, you may also want to set `threadsafe: false`. But as always, make sure you know what you're doing here."